src/components/containers/ModalProvider.js

```diff
@@ -0,0 +1,95 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+
+class ModalProvider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      component: null,
+      componentProps: {},
+      open: false,
+      isAnimatingOut: false,
+    };
+  }
+
+  componentDidUpdate() {
+    const body = document.body;
+    const {open} = this.state;
+
+    // Toggle scroll on document body if modal is open
+    const hasClass = body.classList.contains('no-scroll');
+
```

Comment:
you know, I'm not sure how much is possible, but could we try reducing margins in the modal a little so that there's no need to scroll by default?
It's a little sad to have to scroll everytime I pick a trace type..

Reply:
+1 on faster fades, both in and out please!